risc0/zkp/src/prove/accum.rs

```diff
@@ -21,23 +21,29 @@ use crate::{
     field::{Elem, ExtElem, Field},
 };
 
-/// Tracks plonk accumulations for an execution.
+/// Tracks grand product accumulations for PLOOKUP.
```

Comment:
My understanding was that we are using the PLONK permutation argument to verify that memory is valid. In this section of the code, I'm not sure PLOOKUP applies. I think PLOOKUP is used in the circuit to avoid having to split every bit of a byte or word into an individual column.

Reply:
I'm not sure either -- @jbruestle can you clarify whether Accum is just used for the memory-based grand product accumulation or whether this struct is also involved in the bytes-based accumulation?

Reply:
The answer to the TODO about `kinds` will be clarifying on this front.

Reply:
So both are permutation arguments, but and accum is used for both, PLOOKUP is too specific.  Technically, since at least in our case PLOOKUP is a specialization of PLONK, I'd either call it PLONK, or just a permutation argument.